### PR TITLE
fix(Smash Pros): domain and branding updates

### PR DESCRIPTION
**Pull request by CERLgit**
## Description 

Changed name from Smash Pros to MatchBox, added new domain thematchbox.gg, and updated logos and thumbnails to match current branding.
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Screenshot_20240301_165407

</details>

### CODE REVIEW

File: websites/M/MatchBox/metadata.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"author": {
+		"id": "596877893137596416",
+		"name": "c_fornow"
+	},
+	"service": "MatchBox",
+	"description": {
+		"en": "MatchBox is a skill based matchmaking service for Super Smash Bros. Ultimate"
+	},
+	"url": [
+		"thematchbox.gg",
```
Comment (theusaf) [inline websites/M/MatchBox/metadata.json:12]: Is this an official domain for this service? It currently just shows a landing page

**Review by theusaf — CHANGES_REQUESTED** [review]
Make sure to resolve automated checks!

**Comment by CERLgit** [comment]
> Make sure to resolve automated checks!

The automated check says "Error: Initial version must be 1.0.0".
This is not the initial version its an update to an existing presence updated to match the website's rebrand, should it still be set to 1.0.0?

File: websites/M/MatchBox/metadata.json (hunk 1 above)
Comment (CERLgit) [reply websites/M/MatchBox/metadata.json:12]: https://thematchbox.gg/ works for me, I get a landing page at "www.thematchbox.gg" which is why its not included in the urls

Comment (theusaf) [reply websites/M/MatchBox/metadata.json:12]: Oops, my bad. I didn't realize that I put www in the url bar when testing it...

**Comment by theusaf** [comment]
> > Make sure to resolve automated checks!
> 
> The automated check says "Error: Initial version must be 1.0.0". This is not the initial version its an update to an existing presence updated to match the website's rebrand, should it still be set to 1.0.0?

Yes, this is considered creating a "new" presence and "deleting" an old one.

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**